Source/WebKit/UIProcess/Cocoa/WebProcessPoolCocoa.mm

```diff
@@ -249,6 +250,20 @@ static AccessibilityPreferences accessibilityPreferences()
 }
 #endif
 
+static void logProcessPoolState(const WebProcessPool& pool)
+{
+    for (const auto& process : pool.processes()) {
+        WTF::TextStream stream;
+        stream << process;
```

Comment:
Not sure what TextString does when passing it a `Ref<WebProcessProxy>`. We already print the WebProcessProxy address below, isn't that sufficient?

Reply:
Part of the patch is `WTF::TextStream& operator<<(WTF::TextStream&, const WebProcessProxy&)` which prints a bunch of debugging info about each WebProcessProxy to the given stream. For instance:

```
2022-10-04 11:10:24.775686-0700 0x596d0d   Default     0x0                  88125  0    Safari: (WebKit) [com.apple.WebKit:Process] WebProcessProxy 0x120060000 - pid: 88162, pages: 1, visible-pages: 1, has-service-worker, no-assertion-state, domain: nham.me
```

It seems to work since `Ref<T>` defines an implicit conversion to `T&`.